addons/point_of_sale/static/src/app/utils/input_popups/text_input_with_tag_popup.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+    <t t-name="point_of_sale.TextInputWithTagPopup">
+         <Dialog title="props.title">
+            <t t-foreach="props.buttons" t-as="button" t-key="button_index">
+                <button t-on-click="() => this.buttonClick(button)" type="button" t-attf-class="btn me-2 mb-2 toggle-button {{button.isSelected? 'btn-primary' : 'btn-secondary'}}" >
+                    <t t-esc="button.label"/>
+                </button>
+            </t>
+
+
+            <div class="d-flex flex-wrap p-1 border" t-on-click="onClickTextContainer">
+                 <t t-foreach="state.tagList" t-as="tag" t-key="tag_index">
+                     <div class="ms-1 p-2 badge rounded-pill bg-info text-info bg-opacity-25 d-flex align-items-center" t-esc="tag"></div>
+                 </t>
```

Comment:
can you use the `TagsList` component from web instead ? 
https://github.com/odoo/odoo/blob/181ec327d769dc374363686dd00ae006ebddff11/addons/web/static/src/core/tags_list/tags_list.js#L3